bouncer/src/repo/manipulator/modelconvertor/import/odaHelper/vectorise_device_rvt.h

```diff
@@ -31,27 +31,27 @@ namespace repo {
 				public:
 					VectoriseDeviceRvt();
 
-					void init(GeometryCollector *const geoCollector, MaterialCollectorRvt& matCollector);
+					void init(GeometryCollector *const geoCollector);
 
 					OdGsViewPtr createView(
 						const OdGsClientViewInfo* pInfo = 0,
 						bool bEnableLayerVisibilityPerView = false);
 
 					OdGiConveyorGeometry* destGeometry();
 
+					void setupSimplifier(const OdGiDeviation* pDeviation);
+
 				private:
 					GeometryRvtDumperPtr destGeometryDumper;
-
 					GeometryCollector* geoColl;
-					MaterialCollectorRvt* matColl;
 				};
 
-				class VectorizeView : public OdGsBaseVectorizeViewDef
```

Comment:
Avoid defining 2 classes on a single file(unless the 2nd class is a tiny helper class for the main class) - split this into 2 files.